relay/network_graph/graph.py

```diff
@@ -353,3 +377,22 @@ def mediated_transfer(self, source, target, value):
             target = source
         assert fees == cost
         return cost
+
+    def triangulation_transfer(self, source, target_reduce, target_increase, value):
+        """simulate triangulation transfer off chain"""
+        cost, path = self.find_path_triangulation(source, target_reduce, target_increase, value)
+        assert path[0] == source
+        assert path[-1] == source
+        path = list(reversed(path))
```

Comment:
from here on it is just copy paste. Please refactored and do not duplicate the code. 
For example have a function transfer_path(value, path)